msteams-platform/TOC.yml

```diff
@@ -58,13 +58,13 @@
   items:
   - name: Overview
     href: concepts/design/design-teams-app-overview.md
-  - name: Understand the design process
+  - name: Understand design process
```

Comment:
Removing some preposition and conjunction from the TOC entries is a good idea. It saves valuable screen real-estate and doesn't alter the meaning.

Reply:
Thank you, Ashish.